src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -80,7 +145,15 @@ public RoadAccident getAccidentByIndex7(String index){
 
 
     public RoadAccident getAccidentByIndex(String index){
-        return null;
+        Optional<RoadAccident> result = roadAccidentList.stream()
+                .filter(r -> r.getAccidentId().equals(index))
+                .findFirst();
+
+        if (result.isPresent()) {
```

Comment:
Optional has method orElse(value) that returns "value" if result is not present. so you can solve this task in one line 
return stream.blah.blah.orElse(null);